{{Fudan}}
<html lang="en">
<!--
This html document is created by Tian Huang for Team Fudan iGEM 2018.
We make it compatible on laptop and mobile devices by using Materialize 1.0.0-rc.2.
-->
<head>
    <meta charset="UTF-8">

    <!-- CSS -->
    <link rel="stylesheet" type="text/css" href="http://2018.igem.org/wiki/index.php?title=Template:Fudan/css.css&action=raw&ctype=text/css" />

    <!-- Font-awesome icons 4.7.0 -->
    <link href="http://2018.igem.org/wiki/index.php?title=Template:Fudan/font-awesome.css&action=raw&ctype=text/css" rel="stylesheet">

    <!-- Materialize 1.0.0-rc.2 (Material Design like) -->
    <link rel="stylesheet" href="http://2018.igem.org/wiki/index.php?title=Template:Fudan/materialize.css&action=raw&ctype=text/css">

    <!-- Clear default CSS settings; CSS reset -->
    <style type="text/css">
        *{margin: 0;padding: 0;list-style: none;}
        /* via: https://blog.csdn.net/weixin_41014370/article/details/79523637 */

        /** 清除内外边距 **/
        body, h1, h3, h3, h4, h5, h6, hr, p, blockquote, /* structural elements 结构元素 */
        dl, dt, dd, ul, ol, li, /* list elements 列表元素 */
        pre, /* text formatting elements 文本格式元素 */
        form, fieldset, legend, button, input, textarea, /* form elements 表单元素 */
        th, td /* table elements 表格元素 */ {
            margin: 0;
            padding: 0;
        }

        /** 设置默认字体 **/

        h1, h3, h3, h4, h5, h6 { font-size: 100%; }
        address, cite, dfn, em, var { font-style: normal; } /* 将斜体扶正 */
        code, kbd, pre, samp { font-family: courier new, courier, monospace; } /* 统一等宽字体 */
        small { font-size: 12px; } /* 小于 12px 的中文很难阅读，让 small 正常化 */

        /** 重置列表元素 **/
        ul, ol { list-style: none; }

        /** 重置文本格式元素 **/
        a { text-decoration: none; }
        a:hover { text-decoration: underline; }


        /** 重置表单元素 **/
        legend { color: #000; } /* for ie6 */
        fieldset, img { border: 0; } /* img 搭车：让链接里的 img 无边框 */
        button, input, select, textarea { font-size: 100%; } /* 使得表单元素在 ie 下能继承字体大小 */
        /* 注：optgroup 无法扶正 */

        /** 重置表格元素 **/
        table { border-collapse: collapse; border-spacing: 0; }
        #FudanBody .imgNoM img{
            width:100%;
        }
        #FudanBody .imgNoM{
            text-align:center;
        }
    </style>
</head>

<body>
<!-- Fudan div at igem.org -->
<div id="FudanWrapper" class="white">
    <div id="FudanBody" class="white">
        <header>
            <!-- empty bar -->
            <div id="emptyBar" style="position:relative;width: 100%;"></div>

            <!-- Navigation bar -->
            <nav id="topNav" class="white z-depth-0_5">
                <div class="nav-wrapper">
                    <div id="teamLogo" class="brand-logo">
                        <a href="http://2018.igem.org/Team:Fudan" target="_self"><img src="http://2018.igem.org/wiki/images/c/c4/T--Fudan--teamLogo.svg"></a>
                    </div>
                    <ul id="nav-mobile" class="right orangeBorder">
                        <li class="hide-on-med-and-down"><a class="dropdown-trigger thisPageLink" data-target="dropdown1">Project</a></li>
                        <li class="hide-on-med-and-down"><a class="dropdown-trigger" data-target="dropdown2">Dry lab</a></li>
                        <li class="hide-on-med-and-down"><a class="dropdown-trigger" data-target="dropdown3">Wet lab</a></li>
                        <li class="hide-on-med-and-down"><a class="dropdown-trigger" data-target="dropdown4">Toolbox</a></li>
                        <li class="hide-on-med-and-down"><a class="dropdown-trigger" data-target="dropdown5">Outreach</a></li>
                        <li class="hide-on-med-and-down"><a class="dropdown-trigger" data-target="dropdown6">Team</a></li>
                        <li>
                            <a id="navList" data-target="slide-out" class="waves-effect waves-light sidenav-trigger right">
                                <i class="fa fa-navicon" style="font-size: 24px"></i>
                            </a>
                        </li>
                    </ul>
                </div>
            </nav>

            <!-- Dropdown and List elements in navigation bar -->
            <ul id="dropdown1" class="dropdown-content">
                <li><a href="http://2018.igem.org/Team:Fudan/Demonstrate">Demonstration</a></li>
                <li><a href="http://2018.igem.org/Team:Fudan/Antigen_Receptors">Antigen, Receptors</a></li>
                <li><a href="http://2018.igem.org/Team:Fudan/Results">Transmembrane logic</a></li>
            </ul>
            <ul id="dropdown2" class="dropdown-content">
                <li><a href="http://2018.igem.org/Team:Fudan/Addon#ribo">Addon: ribo</a></li>
                <li><a href="http://2018.igem.org/Team:Fudan/Addon#TALE">Addon: TALE</a></li>
                <li><a href="http://2018.igem.org/Team:Fudan/Addon#T2">Addon: T2</a></li>
                <li><a href="http://2018.igem.org/Team:Fudan/Model#Transcriptional_Amplifer">Model: transcriptional amplifer</a></li>
                <li><a href="http://2018.igem.org/Team:Fudan/Model#War_Predictor">Model: war&nbsp;predictor</a></li>
                <li><a href="http://2018.igem.org/Team:Fudan/Software">Software</a></li>
            </ul>
            <ul id="dropdown3" class="dropdown-content">
                <li><a href="http://2018.igem.org/Team:Fudan/InterLab">iGEM interLab</a></li>
                <li><a href="http://2018.igem.org/Team:Fudan/Notebook">Our notebook</a></li>
                <li><a href="http://2018.igem.org/Team:Fudan/Primers">Primers used</a></li>
                <li><a href="http://2018.igem.org/Team:Fudan/Protocols">Protocols</a></li>
                <li><a href="http://2018.igem.org/Team:Fudan/Safety">Safety</a></li>
            </ul>
            <ul id="dropdown4" class="dropdown-content">
                <li><a href="http://2018.igem.org/Team:Fudan/Basic_Parts">Basic parts</a></li>
                <li><a href="http://2018.igem.org/Team:Fudan/Composite_Parts">Composite parts</a></li>
                <li><a href="http://2018.igem.org/Team:Fudan/Optimization">Optimization</a></li>
                <li><a href="http://2018.igem.org/Team:Fudan/Parts_Collection">Parts collection</a></li>
                <li><a href="http://2018.igem.org/Team:Fudan/Improve">Parts improvement</a></li>
                <li><a href="http://2018.igem.org/Team:Fudan/Measurement">Quantification</a></li>
            </ul>
            <ul id="dropdown5" class="dropdown-content">
                <li><a href="http://2018.igem.org/Team:Fudan/Bio-Art">Bio-Art display</a></li>
                <li><a href="http://2018.igem.org/Team:Fudan/Collaborations">Collaborations</a></li>
                <li><a href="http://2018.igem.org/Team:Fudan/Design_Intention">Design intention</a></li>
                <li><a href="http://2018.igem.org/Team:Fudan/Human_Practices">Human practices</a></li>
                <li><a href="http://2018.igem.org/Team:Fudan/Public_Engagement">Public engagement</a></li>
            </ul>
            <ul id="dropdown6" class="dropdown-content">
                <li><a href="http://2018.igem.org/Team:Fudan/Acknowledgement">Acknowledgement</a></li>
                <li><a href="http://2018.igem.org/Team:Fudan/Attributions">Attributions</a></li>
                <li><a href="http://2018.igem.org/Team:Fudan/Heritage">Heritage</a></li>
                <li><a href="http://2018.igem.org/Team:Fudan/Team">Members</a></li>
                <li><a href="http://2018.igem.org/Team:Fudan/Sponsors">Sponsors</a></li>
            </ul>

            <!-- Slide-out navigator contents -->
            <ul id="slide-out" class="sidenav">
                <li style="padding: 0"><div class="sidenavBanner">
                    <div class="background orangeBg">
                    </div>
                    <p style="width: 100%;text-align: center;font-size: 24px"><span class="white-text">Antigen, Receptors</span></p>
                </div></li>
                <li>
                    <ul class="collapsible expandable">
                        <li>Navigator on wiki</li>
                        <li>
                            <div class="collapsible-header">Project</div>
                            <div class="collapsible-body">
                                <ul>
                                    <li><a href="http://2018.igem.org/Team:Fudan/Demonstrate">Demonstration</a></li>
                                    <li><a href="http://2018.igem.org/Team:Fudan/Antigen_Receptors">Antigen, Receptors</a></li>
                                    <li><a href="http://2018.igem.org/Team:Fudan/Results">Transmembrane logic</a></li>
                                </ul>
                            </div>
                        </li>
                        <li>
                            <div class="collapsible-header">Dry Lab</div>
                            <div class="collapsible-body">
                                <ul>
                                    <li><a href="http://2018.igem.org/Team:Fudan/Addon#ribo">Addon: ribo</a></li>
                                    <li><a href="http://2018.igem.org/Team:Fudan/Addon#TALE">Addon: TALE</a></li>
                                    <li><a href="http://2018.igem.org/Team:Fudan/Addon#T2">Addon: T2</a></li>
                                    <li><a href="http://2018.igem.org/Team:Fudan/Model#Transcriptional_Amplifer">Model: transcriptional amplifer</a></li>
                                    <li><a href="http://2018.igem.org/Team:Fudan/Model#War_Predictor">Model: war&nbsp;predictor</a></li>
                                    <li><a href="http://2018.igem.org/Team:Fudan/Software">Software</a></li>
                                </ul>
                            </div>
                        </li>
                        <li>
                            <div class="collapsible-header">Wet Lab</div>
                            <div class="collapsible-body">
                                <ul>
                                    <li><a href="http://2018.igem.org/Team:Fudan/InterLab">iGEM interLab</a></li>
                                    <li><a href="http://2018.igem.org/Team:Fudan/Notebook">Our notebook</a></li>
                                    <li><a href="http://2018.igem.org/Team:Fudan/Primers">Primers used</a></li>
                                    <li><a href="http://2018.igem.org/Team:Fudan/Protocols">Protocols</a></li>
                                    <li><a href="http://2018.igem.org/Team:Fudan/Safety">Safety</a></li>
                                </ul>
                            </div>
                        </li>
                        <li>
                            <div class="collapsible-header">Toolbox</div>
                            <div class="collapsible-body">
                                <ul>
                                    <li><a href="http://2018.igem.org/Team:Fudan/Basic_Parts">Basic parts</a></li>
                                    <li><a href="http://2018.igem.org/Team:Fudan/Composite_Parts">Composite parts</a></li>
                                    <li><a href="http://2018.igem.org/Team:Fudan/Optimization">Optimization</a></li>
                                    <li><a href="http://2018.igem.org/Team:Fudan/Parts_Collection">Parts collection</a></li>
                                    <li><a href="http://2018.igem.org/Team:Fudan/Improve">Parts improvement</a></li>
                                    <li><a href="http://2018.igem.org/Team:Fudan/Measurement">Quantification</a></li>
                                </ul>
                            </div>
                        </li>
                        <li>
                            <div class="collapsible-header">Outreach</div>
                            <div class="collapsible-body">
                                <ul>
                                    <li><a href="http://2018.igem.org/Team:Fudan/Bio-Art">Bio-Art display</a></li>
                                    <li><a href="http://2018.igem.org/Team:Fudan/Collaborations">Collaborations</a></li>
                                    <li><a href="http://2018.igem.org/Team:Fudan/Design_Intention">Design intention</a></li>
                                    <li><a href="http://2018.igem.org/Team:Fudan/Human_Practices">Human practices</a></li>
                                    <li><a href="http://2018.igem.org/Team:Fudan/Public_Engagement">Public engagement</a></li>
                                </ul>
                            </div>
                        </li>
                        <li>
                            <div class="collapsible-header">Team</div>
                            <div class="collapsible-body">
                                <ul>
                                    <li><a href="http://2018.igem.org/Team:Fudan/Acknowledgement">Acknowledgement</a></li>
                                    <li><a href="http://2018.igem.org/Team:Fudan/Attributions">Attributions</a></li>
                                    <li><a href="http://2018.igem.org/Team:Fudan/Heritage">Heritage</a></li>
                                    <li><a href="http://2018.igem.org/Team:Fudan/Team">Members</a></li>
                                    <li><a href="http://2018.igem.org/Team:Fudan/Sponsors">Sponsors</a></li>
                                </ul>
                            </div>
                        </li>
                    </ul>

                </li>
                <li><div class="placeHolder"></div></li>
            </ul>
        </header>

        <div id="pageContent" style="">


            <div id="contentBanner" class="figureBanner orangeBg">
                <div class="row">
                    <div class="col s12 hide-on-med-and-up">
                        <h1>Antigen, Receptors</h1>
                    </div>
                    <div class="col s12 hide-on-med-and-up">
                        <span></span>
                    </div>
                </div>
                <div id="figureBannerTitle" class="hide-on-small-only">
                    <h1>Antigen, Receptors</h1>
                    <p><span></span></p>
                </div>
                <div class="hide-on-small-only">
                    <img src="http://2018.igem.org/wiki/images/c/c2/T--Fudan--title_researchbackground.jpg">
                    <svg width="10" height="10" xmlns="http://www.w3.org/2000/svg" style="position:absolute; left:0;top:0; width: 4%;height: 100%;">
                        <defs>
                            <linearGradient y2="0%" x2="100%" y1="0%" x1="0%" id="blackgraleft">
                                <stop stop-color="rgb(0,0,0)" stop-opacity="1" offset="0%"/>
                                <stop stop-color="rgb(0,0,0)" stop-opacity="0" offset="100%"/>
                            </linearGradient>
                        </defs>
                        <g>
                            <rect id="svg_1" fill="url(#blackgraleft)" height="100%" width="100%"/>
                        </g>
                    </svg>
                    <svg width="10" height="10" xmlns="http://www.w3.org/2000/svg" style="position:absolute; right:0;top:0; width: 4%;height: 100%;">
                        <defs>
                            <linearGradient y2="0%" x2="100%" y1="0%" x1="0%" id="blackgraright">
                                <stop stop-color="rgb(0,0,0)" stop-opacity="0" offset="0%"/>
                                <stop stop-color="rgb(0,0,0)" stop-opacity="1" offset="100%"/>
                            </linearGradient>
                        </defs>
                        <g>
                            <rect id="svg_2" fill="url(#blackgraright)" height="100%" width="100%"/>
                        </g>
                    </svg>
                </div>
            </div>

            <!-- @@@@ main content of the page -->
            <div class="container">
                <main style="margin: 0">
                    <div id="section1" class="section container scrollspy">
                        <h2>Signal transduction system review and comparison
                        </h2>
                        <div class="expFigureHolder" style="width:100%">
                            <img class="responsive-img" src="http://2018.igem.org/wiki/images/8/88/T--Fudan--aa1.svg">

                        </div>
                        <p>
                            In order to be able to implement a custom multiplexed transmembrane signal input/output relationship, the first condition is that engineering modular receptor to enable it to recognize extracellular signals and transduce them into customized intracellular signals. To this end, a variety of techniques have been developed, such as Tango(Barnea et al., 2008), CAR(Porter et al., 2011), GEMS(Scheller et al., 2018), MESA(Daringer et al., 2014; Hartfield et al., 2017; Schwarz et al., 2017), SynNotch(Morsut et al., 2016; Roybal et al., 2016a; Roybal et al., 2016b; Toda et al., 2018), etc. High programmability of the extracellular and intracellular domains of SynNotch, as well as adaptation to contact-dependent signaling, fully meets our needs. Thus we ultimately apply SynNtoch technology as a receiving port for extracellular signals of transmembrane logic gates.
   </p> <p>
Notch receptors are single-pass transmembrane receptors that transduce extracellular surface signals from sender cells to the inside of receiver cells. It features structural modularity and orthogonality in intracellular signal transduction. The signal strength and intensity of wild-type Notch receptors plays a crucial role in cell-fate decisions such as proliferation in development as well as homeostasis. This is true for functional outcomes that mainly rely on the dosage of the gene regulation and context of Notch signaling (Aster et al., 2008; Bray, 2016) For SynNotch, specialized factors will exclusively participate in regulation of genetic circuits that allow novel user-defined transcriptional responses. In our project ENABLE, we have engineered the cells to perform logical operations on multiple transmembrane signals to produce a variety of customizable outputs

                        </p>
                        <h3>What is different in signal input?
                        </h3>
                        <h4>1.	Ligands
                        </h4>
                        <p>
                            The largest family of ligands that can activate wild type Notch receptors features three related structural motifs, which are DSL (Delta/Serrate/LAG-2) motif, EGF-like repeats, and DOS (Delta and OSM-11-like proteins) domain that are tandem EGF repeats. (Kopan et al., 2009) Wild type Notch receptors also features cis-inhibition and trans-activation due to the antiparallel orientation of extracellular domains.
                         </p><p>           With SynNotch receptors, the cis-inhibition mechanism was recapitulated in Morsut et al., 2016. This indicates that the activating and inhibiting mechanism of wild type Notch and SynNotch are probably alike in nature. Therefore, as long as the Notch core can be activated, any ectodomain could work. In 2015, Gordon and her colleagues demonstrated that Notch with FBP as ectodomain targeting FKBP in the presence of rapamycin can activate Notch receptors. (Gordon et al., 2015) Lim and his colleagues also showed SynNotch can utilize novel antigens such as CD19 and EGFP.


                        </p>
                        <h4>2.	The ectodomain of Notch receptors
                        </h4>
                        <p>
                           Previous work by Morsut et al. showed that the natural surface antigen CD19 and the non-natural surface antigen EGFP have good orthogonality(Toda et al., 2018). We applied this and designed two surface antigens, surCD19 and surEGFP. By immunostaining, we were able to clearly detect surCD19 and surEGFP expressed on 293T cells. Thus, these two surface antigens can serve as ideal dual antigens for our transmembrane binary Boolean logic inputs.
                        </p><p>       The ectodomain of wild type Notch is composed of 29-36 modular epidermal growth factor (EGF)-like repeats, each around 40 residues in length and can be categorized into calcium binders and non-binders. Though former studies have assumed the shape of this ectodomain to be a long rod-like structure, taking a linear topology (Kopan and Ilagna,2009), recent studies have elucidated that there are multiple rods joined by flexible joints. (Weisshuhn et al., 2015a, Weisshuhn et al., 2015b)

                        </p>
                        <h3>
                            Signal transduction of the Notch core is the same in theory

                        </h3>
                        <p>The signal transduction mechanism of Notch signaling is very unique. While conserved signaling pathways are activated at the efforts of receptors, Notch receptor signaling and proteolysis is caused by the action of corresponding ligands.
                        </p><p>    Another interesting feature of Notch signaling is that its intracellular domains, released by intramembrane proteolysis during activation, will translocate to the cell nucleus. (Schroeter EH et al., 1998) This renders almost no amplification of the signal in this layer, and more importantly, signal orthogonality.
                    </p><p>    Notch specific ligands are in constant endocytosis-surface presenting cycle in the signal-sender cell. When the N terminal recognition domain of Notch is bound to the ligand, this would produce resistance that will stimulate the ubiquitylation of the ligand and also enhance the recruitment of ubiquitin-binding epsin endocytic adaptors (Weinmaster G. 2011). Complementary to this model, Ligand and cells defective in endocytosis cannot activate Notch. (Nichols JT et al., 2007) This is like a cellular level “tug-of-war” between the two surface proteins, with the Notch receptor eventually giving in. When Notch extracellular domain is not bound by ligands with enough affinity to it or is under disturbances from small molecules, Notch 1 receptor will not activate because its negative regulatory region (NRR) will refrain it from happening.
                        </p>
                        <div class="expFigureHolder" style="width:40%;margin: 0 auto;">
                            <img class="responsive-img" src="http://2018.igem.org/wiki/images/f/f1/T--Fudan--bestpart2.png">
<p>
    Figure1 : Structure of the negative regulatory region of the human Notch 1 receptor.<br>
    Adapted from Gordon et al., 2009 (A and B paper), using the PDB 3eto deposited by them.
</p>
                        </div>
                        <p>
                            Notch core consists of the negatively regulated region and transmembrane region of the mouse Notch 1 receptor. It is the joint that empowers SynNotch the high modularity and proper activation. From the N terminal to C terminal, it is composed of three lin12-repeats (LNR) domains LNR-A, LNR-B, LNR-C, the heterodimerization domain (HD), and the transmembrane domain. The LNR-AB linker between LNR-A and B is like a plug that occludes ADAM from cleavts wild type extracellular domain can be substituted by antiGFP scfvs like LaG16 and anti-CD19, while its intracellular domain can be replaced by specialized transcription factors. Notch core is also the key to the activation of both wild type Notch receptors and SynNotch.Lin-12 repeats (LNR) (Vardar et al. 2003)
                        </p><p>             The majority of Notch receptors are cleaved during the secretory pathway by furin-like convertase at cleavage site 1 (S1) (Logeat 1998), which protrudes as a part of a linker of the two parts of the Notch negative regulatory region's heterodimerization domain. (Gordon et al., 2009 A; Gordon et al., 2007) This action separates Notch extracellular domain and the Notch transmembrane domain on the level of peptide linkage, but noncovalent interactions between the lose heterodimerization domain holds the receptor together. Despite disparate results on the function of S1 cleavage is a prerequisite for later activation, it has also been suggested that it is not functional required for contact-dependent activation or expression on the cell membrane. (Gordon et al., 2009 B)
                    </p><p>           When ligand binds to the Notch extracellular domain (NECD) and pulls it with mechanical force and thereby activating it, this action initiates a process called regulated intramembrane proteolysis (RIP). (Weinmaster et al., 1997, Brou et al. 2000, Gordon et al., 2015) In this process, the Notch receptor is first cleaved by two types of metalloproteases in the ADAM family (a disintegrin and metalloprotease), ADAM10/17, while ADAM10 is the dominant one. (Brou et al.,2000; Mumm et al., 2000, Bozkulak et al., 2009). The truncated transmembrane subunit remaining will be subsequently cleaved by  gamma-secretase, releasing the Notch intracellular domain (NICD). The wild type NICD will then enter the nucleus and interact with CSL and MAML (Schroteter et al., 1998, Struhl et al., 1998)

                        </p>
                        <h3>What is different in transcriptional signal output?
                        </h3>
                        <p>
                            The wild type NICD will enter the nucleus and interact with CSL and MAML (Schroteter et al., 1998, Struhl et al., 1998) to regulate endogenous pathways that decide cell fate. SynNotch receptor’s customizable output have been integrated with chimeric antigen receptors to engineer T cells with customizable therapeutic response programs (Roybal et al., 2016 a and b ). programming self-organizing multicellular structures (Toda et al., 2018) In our project ENABLE, we have combined two types of SynNotch in order to empower mammalian cell the ability of binary computation. This promises more novel application potentials.


                        </p>
                    </div>
                    <div class="section container">
                        <h4 style="margin-left: 0;">References</h4>
                        <ol id="ref" style="margin: 23px 0 0 0;">
                            <li>Aster J C, Pear W S, Blacklow S C. Notch signaling in leukemia[J]. Annu. Rev. pathmechdis. Mech. Dis., 2008, 3: 587-613.
                            </li><li>
                                Bozkulak, Esra Cagavi, and Gerry Weinmaster. "Selective use of ADAM10 and ADAM17 in activation of Notch1 signaling." Molecular and cellular biology 29.21 (2009): 5679-5695.

                        </li><li>     Bray, Sarah J. "Notch signalling in context." Nature reviews Molecular cell biology 17.11 (2016): 722.

                        </li><li>     Brou C, Logeat F, Gupta N, Bessia C, LeBail O, et al. (2000) A novel proteolytic cleavage involved in Notch signaling: the role of the disintegrin-metalloprotease TACE. Mol Cell 5: 207–216.

                        </li><li>    Chiba S. Concise review: Notch signaling in stem cell systems[J]. Stem cells, 2006, 24(11): 2437-2447.

                        </li><li>    Fehon, R.G. et al. Molecular interactions between the protein products of the neurogenic loci Notch and Delta, two EGF-homologous genes in Drosophila. Cell 61, 523–534 (1990).

                        </li><li>   Fridy, P. C. et al. A robust pipeline for rapid production of versatile nanobody repertoires. Nature methods 11, 1253--1260 (2014).

                        </li><li>    Gordon WR, Roy M, Vardar-Ulu D, Garfinkel M, Mansour MR, et al. (2009) Structure of the Notch1-negative regulatory region: implications for normal activation and pathogenic signaling in T-ALL. Blood 113: 4381–4390.

                        </li><li>   Gordon WR, Vardar-Ulu D, Histen G, Sanchez-Irizarry C, Aster JC, et al.(2007) Structural basis for autoinhibition of Notch. Nat Struct Mol Biol 14: 295–300.

                        </li><li>    A: Gordon, Wendy R., et al. "Structure of the Notch1-negative regulatory region: implications for normal activation and pathogenic signaling in T-ALL." Blood 113.18 (2009): 4381-4390.
                        </li><li>     B: Gordon, Wendy R., et al. "Effects of S1 cleavage on the structure, surface export, and signaling activity of human Notch1 and Notch2." PloS one 4.8 (2009): e6613.

                        </li><li>   Gordon, Wendy R., et al. "Mechanical allostery: evidence for a force requirement in the proteolytic activation of Notch." Developmental cell 33.6 (2015): 729-736.

                        </li><li>   Greenwald, Iva, and Geraldine Seydoux. "Analysis of gain-of-function mutations of the lin-12 gene of Caenorhabditis elegans." Nature 346.6280 (1990): 197.

                        </li><li>  Grupp, S. A. et al. Chimeric antigen receptor-modified T cells for acute lymphoid leukemia. New England Journal of Medicine 368, 1509--1518 (2013).

                        </li><li>   Kovall R A, Gebelein B, Sprinzak D, et al. The canonical Notch signaling pathway: structural and biochemical insights into shape, sugar, and force[J]. Developmental cell, 2017, 41(3): 228-241.

                        </li><li>     Kopan, R., and Ilagan, M.X. (2009). The canonical notch signaling pathway: unfolding the activation mechanism. Cell 137, 216–233.

                        </li><li>     Logeat, F. et al. The Notch1 receptor is cleaved constitutively by a furin-like convertase. Proceedings of the National Academy of Sciences 95, 8108-8112 (1998).

                        </li><li>  Mansour, M. R., et al. "High incidence of Notch-1 mutations in adult patients with T-cell acute lymphoblastic leukemia." Leukemia 20.3 (2006): 537.

                        </li><li>   Mumm J S, Kopan R. Notch signaling: from the outside in[J]. Developmental biology, 2000, 228(2): 151-16

                        </li><li>   Mumm JS, Schroeter EH, Saxena MT, Griesemer A, Tian X, et al. (2000) A ligand-induced extracellular cleavage regulates gamma-secretase-like proteolytic activation of Notch1. Mol Cell 5: 197–206.

                        </li><li>    Nichols JT, Miyamoto A, Olsen SL, D’Souza B, Yao C, Weinmaster G. DSL ligand endocytosis physically dissociates Notch1 heterodimers before activating proteolysis can occur. J Cell Biol 2007;176:445–58.


                        </li><li>   Rand, Matthew D., et al. "Calcium depletion dissociates and activates heterodimeric notch receptors." Molecular and cellular biology 20.5 (2000): 1825-1835.

                        </li><li>   Sanchez-Irizarry, Cheryll, et al. "Notch subunit heterodimerization and prevention of ligand-independent proteolytic activation depend, respectively, on a novel domain and the LNR repeats." Molecular and cellular biology 24.21 (2004): 9265-9273.

                        </li><li>   Schroeter EH, Kisslinger JA, Kopan R. Notch-1 signalling requires ligand-induced proteolytic re- lease of intracellular domain. Nature. 1998;393: 382-386.

                        </li><li>   Struhl, G. & Adachi, A. Nuclear access and action of notch in vivo. Cell 93, 649–660 (1998).

                        </li><li>   Vardar, Didem, et al. "Nuclear magnetic resonance structure of a prototype Lin12-Notch repeat module from human Notch1." Biochemistry 42.23 (2003): 7061-7067.

                        </li><li>   Weinmaster, G. The ins and outs of notch signaling. Mol. Cell. Neuroscience. 9, 91–102 (1997).
                        </li><li>   Weinmaster G, Fischer JA. Notch ligand ubiquitylation: what is it good for. Dev Cell 2011;21:134–44.

                        </li><li>   Weisshuhn, P.C., Handford, P.A., and Redﬁeld, C. (2015a). (1)H, (13)C and (15)N assignments of EGF domains 4 to 7 of human Notch-1. Biomol. NMR Assign. 9, 275–279.

                        </li><li>    Weisshuhn, P.C., Handford, P.A., and Redﬁeld, C. (2015b). (1)H, (13)C and (15)N assignments of EGF domains 8-11 of human Notch-1. Biomol. NMR Assign. 9, 375–379.

                        </li><li>   Weisshuhn, P.C., Sheppard, D., Taylor, P., Whiteman, P., Lea, S.M., Handford, P.A., and Redﬁeld, C. (2016). Non-linear and ﬂexible regions of the human Notch1 extracellular domain revealed by high-resolution structural studies. Structure 24, 555–566.
                        </li><li>
                                Cho J H, Okuma A, Al-Rubaye D, et al. Engineering Axl specific CAR and SynNotch receptor for cancer therapy[J]. Scientific reports, 2018, 8(1): 3846.

                        </li><li>  Morsut L, Roybal K T, Xiong X, et al. Engineering customized cell sensing and response behaviors using synthetic notch receptors[J]. Cell, 2016, 164(4): 780-791.
                                 
                        </li><li>      Roybal K T, Williams J Z, Morsut L, et al. Engineering T cells with customized therapeutic response programs using synthetic notch receptors[J]. Cell, 2016, 167(2): 419-432. e16.
                                 
                        </li><li>     Roybal K T, Rupp L J, Morsut L, et al. Precision tumor recognition by T cells with combinatorial antigen-sensing circuits[J]. Cell, 2016, 164(4): 770-779.
                                 
                        </li><li>     Toda S, Blauch L R, Tang S K Y, et al. Programming self-organizing multicellular structures with synthetic cell-cell signaling[J]. Science, 2018: eaat0271.

                            </li>
                        </ol>
                    </div>
                </main>
            </div>
            <!-- @@@@ end of main content of the page -->

            <!--Abstract on content page-->
            <div id="abstractContent" class="z-depth-2">
                <a href="#!"><img src="http://2018.igem.org/wiki/images/9/96/T--Fudan--X.svg"></a>
                <div class="container">
                    <h2 style="margin: 0;padding: 10px 0;">Abstract</h2>
                    <p style="margin: 0">Contact-dependent signaling is critical for multicellular biological
                        events, yet customizing contact-dependent signal transduction between
                        cells remains challenging. Here we have developed the ENABLE toolbox, a
                        complete set of transmembrane binary logic gates. Each gate consists of
                        3 layers: Receptor, Amplifier, and Combiner. We first optimized synthetic
                        Notch receptors to enable cells to respond to different signals across the
                        membrane reliably. These signals, individually amplified intracellularly by
                        transcription, are further combined for computing. Our engineered zinc finger-based
                        transcription factors perform binary computation and output designed products.
                        In summary, we have combined spatially different signals in mammalian cells,
                        and revealed new potentials for biological oscillators, tissue engineering,
                        cancer treatments, bio-computing, etc. ENABLE is a toolbox for constructing
                        contact-dependent signaling networks in mammals. The 3-layer design principle
                        underlying ENABLE empowers any future development of transmembrane logic circuits,
                        thus contributes a foundational advance to Synthetic Biology.
                    </p>
                </div>
            </div>

            <!-- Floating Btns -->
            <div class="floatingBtn">
                <a href="#!" id="abstractBtn" class="btn">
                    <i class="fa fa-sticky-note" style="font-size: 30px;line-height: 50px"></i>
                </a>
                <a href="#FudanWrapper" class="btn">
                    <i class="fa fa-angle-up" style="font-size: 48px;line-height: 45px"></i>
                </a>
            </div>

            <!-- Footer with sponsors and contact methods -->
            <footer id="FudanFooter" class="page-footer grey">
                <div class="container">
                    <div class="row">
                        <div id="sponsor" class="col m3 s12 row">
                            <a href="http://2018.igem.org/Team:Fudan" target="_blank"><img class="col s3 m6 l3" style="position:relative; padding: 0 0.3rem; margin:-0.15rem 0; left: -0.45rem;" src="http://2018.igem.org/wiki/images/7/73/T--Fudan--teamLogoWhite.png">
                            </a><a href="http://www.fudan.edu.cn/en/" target="_blank"><img class="col s3 m6 l3" alt="Fudan University" src="http://2018.igem.org/wiki/images/f/f7/T--Fudan--schoolLogo.png">
                        </a><a href="http://life.fudan.edu.cn/" target="_blank"><img class="col s3 m6 l3" style="margin-bottom: 4%;/* 该图比其他小一点，排版需要 */" alt="School of Life Sciences, Fudan University" src="http://2018.igem.org/wiki/images/1/1d/T--Fudan--schoolOfLifeSciencesIcon.png">
                        </a><a href="http://www.yfc.cn/en/" target="_blank"><img class="col s3 m6 l3" style="padding: 0.15rem 0.9rem;" alt="Yunfeng Capital" src="http://2018.igem.org/wiki/images/e/e2/T--Fudan--yunfengLogo.png">
                        </a>
                            <h3 class="col s12" style="text-align: left; color: rgba(255, 255, 255, 0.8); font-size: 18px">ENABLE: making cells even smarter</h3>
                        </div>
                        <div id="usefulLinks" class="col m9 s12 row">
                            <div class="col s12 l6 row">
                                <div class="col s12 m4 active">
                                    <span>Project</span>
                                    <ul>
                                        <li><a href="http://2018.igem.org/Team:Fudan/Demonstrate">Demonstration</a></li>

                                        <li><a href="http://2018.igem.org/Team:Fudan/Antigen_Receptors">Antigen, Receptors</a></li>
                                        <li><a href="http://2018.igem.org/Team:Fudan/Results">Transmembrane logic</a></li>
                                        <li><a href="http://2017.igem.org/Team:Fudan">2017.iGEM</a></li>
                                    </ul>
                                </div>
                                <div class="col s12 m4">
                                    <span>Dry lab</span>
                                    <ul>
                                        <li><a href="http://2018.igem.org/Team:Fudan/Addon#ribo">Addon: ribo</a></li>
                                        <li><a href="http://2018.igem.org/Team:Fudan/Addon#TALE">Addon: TALE</a></li>
                                        <li><a href="http://2018.igem.org/Team:Fudan/Addon#T2">Addon: T2</a></li>
                                        <li><a href="http://2018.igem.org/Team:Fudan/Model#Transcriptional_Amplifer">Model: transcriptional amplifer</a></li>
                                        <li><a href="http://2018.igem.org/Team:Fudan/Model#War_Predictor">Model: war&nbsp;predictor</a></li>
                                        <li><a href="http://2018.igem.org/Team:Fudan/Software">Software</a></li>
                                    </ul>
                                </div>
                                <div class="col s12 m4">
                                    <span>Wet lab</span>
                                    <ul>
                                        <li><a href="http://2018.igem.org/Team:Fudan/InterLab">iGEM interLab</a></li>
                                        <li><a href="http://2018.igem.org/Team:Fudan/Notebook">Our notebook</a></li>
                                        <li><a href="http://2018.igem.org/Team:Fudan/Primers">Primers used</a></li>
                                        <li><a href="http://2018.igem.org/Team:Fudan/Protocols">Protocols</a></li>
                                        <li><a href="http://2018.igem.org/Team:Fudan/Safety">Safety</a></li>
                                    </ul>
                                </div>
                            </div>
                            <div class="col s12 l6 row">
                                <div class="col s12 m4">
                                    <span>Toolbox</span>
                                    <ul>
                                        <li><a href="http://2018.igem.org/Team:Fudan/Basic_Parts">Basic parts</a></li>
                                        <li><a href="http://2018.igem.org/Team:Fudan/Composite_Parts">Composite parts</a></li>
                                        <li><a href="http://2018.igem.org/Team:Fudan/Optimization">Optimization</a></li>
                                        <li><a href="http://2018.igem.org/Team:Fudan/Parts_Collection">Parts collection</a></li>
                                        <li><a href="http://2018.igem.org/Team:Fudan/Improve">Parts improvement</a></li>
                                        <li><a href="http://2018.igem.org/Team:Fudan/Measurement">Quantification</a></li>
                                    </ul>
                                </div>
                                <div class="col s12 m4">
                                    <span>Outreach</span>
                                    <ul>
                                        <li><a href="http://2018.igem.org/Team:Fudan/Bio-Art">Bio-Art display</a></li>
                                        <li><a href="http://2018.igem.org/Team:Fudan/Collaborations">Collaborations</a></li>
                                        <li><a href="http://2018.igem.org/Team:Fudan/Design_Intention">Design intention</a></li>
                                        <li><a href="http://2018.igem.org/Team:Fudan/Human_Practices">Human practices</a></li>
                                        <li><a href="http://2018.igem.org/Team:Fudan/Public_Engagement">Public engagement</a></li>
                                    </ul>
                                </div>
                                <div class="col s12 m4">
                                    <span>Team</span>
                                    <ul>
                                        <li><a href="http://2018.igem.org/Team:Fudan/Acknowledgement">Acknowledgement</a></li>
                                        <li><a href="http://2018.igem.org/Team:Fudan/Attributions">Attributions</a></li>
                                        <li><a href="http://2018.igem.org/Team:Fudan/Heritage">Heritage</a></li>
                                        <li><a href="http://2018.igem.org/Team:Fudan/Team">Members</a></li>
                                        <li><a href="http://2018.igem.org/Team:Fudan/Sponsors">Sponsors</a></li>
                                    </ul>
                                </div>
                            </div>
                        </div>
                    </div>
                </div>
                <div class="footer-copyright">
                    <div class="container">
                        <div class="contactUS row">
                            <div class="col s12 m6 l4"><i class="fa fa-location-arrow"></i> Life Sci Bldg E301, 2005 Songhu Rd, Shanghai
                            </div><div class="col s12 m6 l3"><i class="fa fa-envelope-o"></i> igem@fudan.edu.cn
                        </div><div class="col s12 m6 l2"><i class="fa fa-wechat"></i> Fudan_iGEM
                        </div><div class="col s12 m6 l3"><i class="fa fa-fax"></i> +86-21-31246727
                        </div>
                        </div>
                    </div>
                </div>
            </footer>

        </div>
    </div>
</div>


<!-- Javascript files -->
<!-- Materialize 1.0.0-rc.2 -->
<script src="http://2018.igem.org/wiki/index.php?title=Template:Fudan/materialize.js&action=raw&ctype=text/javascript"></script>

<!-- Javascript -->
<script src="http://2018.igem.org/wiki/index.php?title=Template:Fudan/js.js&action=raw&ctype=text/javascript"></script>

</body>
</html>