{{Fudan-TSI}}<script src="https://ajax.aspnetcdn.com/ajax/jQuery/jquery-1.11.3.min.js"></script>
<html></p></div></div></div><meta name="viewport" content="width=device-width, initial-scale=1"><meta charset="UTF-8">
  <link rel="stylesheet" href="https://2019.igem.org/wiki/index.php?title=Template:Fudan-TSI/materialize.css&action=raw&ctype=text/css">
  <link rel="stylesheet" href="https://2019.igem.org/wiki/index.php?title=Template:Fudan-TSI/Fudan-font-awesome.css&action=raw&ctype=text/css" />
  <link rel="stylesheet" type="text/css" href="https://2019.igem.org/wiki/index.php?title=Template:Fudan-TSI/Fudan-css.css&action=raw&ctype=text/css" />
<script>
////////////////////////////////////////////////////////////////////////////////
function highlight_current_page_menu(){var e="https://2019.igem.org/";e+=wgPageName,$("a[href$='"+e+"']").children().addClass("current_page"),$(".current_page").hasClass("submenu_item")&&($(".current_page").parent().parent().fadeToggle(400),$(".current_page").parent().parent().prev().addClass("current_page"),$(".menu_item.current_page > .submenu_control_icon").toggleClass("open"))}$(document).ready(function(){$("#HQ_page").attr("id",""),highlight_current_page_menu(),$(".menu_item").click(function(){$(".submenu_control_icon",this).toggleClass("open"),$(this).next(".submenu").fadeToggle(400)}),$(".igem_2019_team_mobile_bar").click(function(){$(this).next().toggleClass("displaying_menu")})});
////////////////////////////////////////////////////////////////////////////////
</script><style>
/*****************************************************************************/
/* DEFAULT WIKI SETTINGS */
/*****************************************************************************/
  #home_logo, #sideMenu { display:none; }
  #sideMenu, #top_title, .patrollink { display:none; }
  #content { margin-left: 0; padding:0px; width:100%; }
  .judges-will-not-evaluate { border: 4px solid #e4dede; padding: 2% !important; width: 92%!important; }
  *{margin:0;padding:0;list-style:none}blockquote,body,button,dd,dl,dt,fieldset,form,h1,h3,h4,h5,h6,hr,input,legend,li,ol,p,pre,td,textarea,th,ul{margin:0;padding:0}address,cite,dfn,em,var{font-style:normal}code,kbd,pre,samp{font-family:courier new,courier,monospace}ol,ul{list-style:none}a{text-decoration:none}a:hover{text-decoration:underline}legend{color:#000}fieldset,img{border:0}button,input,select,textarea{font-size:100%}table{border-collapse:collapse;border-spacing:0}
    </style>
    <title>2019 Team:Fudan-TSI Design</title>
</head>
<body>
<div id="FudanTSIdivWrapper"><div id="FudanTSIBody">
  <header>
  <div id="emptyBar" style="position:relative;width: 100%;"></div><nav id="topNav" class="black z-depth-0_5"><div class="nav-wrapper"><div id="teamLogo" class="brand-logo"> <a href="/Team:Fudan-TSI" target="_self"><img alt="2019 team logo" src="https://2019.igem.org/wiki/images/d/d3/T--Fudan-TSI--HomepageLogo.gif"></a></div><ul id="nav-mobile" class="right">
    <li> <a id="navList" data-target="slide-out" class="waves-effect waves-light sidenav-trigger right"> <i class="fa fa-navicon" style="font-size: 24px"></i> </a></li></ul></div> </nav>
  <!-- Dropdown and List elements in navigation bar -->

  <ul id="slide-out" class="sidenav">
    <li style="padding: 0"><div class="sidenavBanner">
      <div class="background"></div>
      <p class="flow-text" style="width:100%;text-align:center"><span class="white-text">Design</span></p>
    </div></li>
    <li>
      <ul class="collapsible expandable">
        <li class="onThisPageNav"><span>On this page</span></li>
        <li class="onThisPageNav"><a href="#section1">MMLV-RT</a></li>
        <li class="onThisPageNav"><a href="#section2">tRNA primer</a></li>
        <li class="onThisPageNav"><a href="#section3">Flanking sequences</a></li>
        <li class="onThisPageNav"><a href="#section4">Cre recombinase</a></li>
        <li class="onThisPageNav"><a href="#section5">Experiments</a></li>
        <li class="onThisPageNav"><a href="#section6">Testing</a></li>
        <li class="onThisPageNav"><a href="/Team:Fudan-TSI/Applied_Design">Applied design</a></li>
        <li class="onThisPageNav"><a href="#section7">References</a></li>

        <li><span class="pageSidebar">Team: Fudan-TSI</span></li><li><div class="collapsible-header active"><span class="pageSidebar">Project</span></div><div class="collapsible-body"><ul><li><a class="pageSidebar" href="/Team:Fudan-TSI/Description">Background</a></li><li><a class="pageSidebar" href="/Team:Fudan-TSI/Design">Design</a></li><li><a class="pageSidebar" href="/Team:Fudan-TSI/Experiments">Experiments</a></li><li><a class="pageSidebar" href="/Team:Fudan-TSI/Applied_Design">Applied design</a></li><li><a class="pageSidebar" href="/Team:Fudan-TSI/Judging">Judging</a></li></ul></div></li><li><div class="collapsible-header"><span class="pageSidebar">Results</span></div><div class="collapsible-body"><ul><li><a class="pageSidebar" href="/Team:Fudan-TSI/Demonstrate#ReverseTranscription">Reverse transcription</a></li><li><a class="pageSidebar" href="/Team:Fudan-TSI/Demonstrate#Recombination">Recombination</a></li><li><a class="pageSidebar" href="/Team:Fudan-TSI/Demonstrate">Demonstration</a></li><li><a class="pageSidebar" href="/Team:Fudan-TSI/Measurement">Measurement</a></li><li><a class="pageSidebar" href="/Team:Fudan-TSI/Notebook">Notebook</a></li></ul></div></li><li><div class="collapsible-header"><span class="pageSidebar">Model</span></div><div class="collapsible-body"><ul><li><a class="pageSidebar" href="/Team:Fudan-TSI/Model">Modeling</a></li><li><a class="pageSidebar" href="/Team:Fudan-TSI/Software">Software</a></li><li><a class="pageSidebar" href="/Team:Fudan-TSI/Hardware">Hardware</a></li></ul></div></li><li><div class="collapsible-header"><span class="pageSidebar">Parts</span></div><div class="collapsible-body"><ul><li><a class="pageSidebar" href="/Team:Fudan-TSI/Basic_Part">Basic parts</a></li><li><a class="pageSidebar" href="/Team:Fudan-TSI/Composite_Part">Composite parts</a></li><li><a class="pageSidebar" href="/Team:Fudan-TSI/Improve">Part improvement</a></li><li><a class="pageSidebar" href="/Team:Fudan-TSI/Part_Collection">Part collection</a></li></ul></div></li><li><div class="collapsible-header"><span class="pageSidebar">Human practices</span></div><div class="collapsible-body"><ul><li><a class="pageSidebar" href="/Team:Fudan-TSI/Public_Engagement">Public engagement</a></li><li><a class="pageSidebar" href="/Team:Fudan-TSI/Human_Practices#IntegratedHumanPractice">Integrated HP</a></li><li><a class="pageSidebar" href="/Team:Fudan-TSI/Collaborations">Collaborations</a></li><li><a class="pageSidebar" href="/Team:Fudan-TSI/Safety">Safety</a></li></ul></div></li><li><div class="collapsible-header"><span class="pageSidebar">Team</span></div><div class="collapsible-body"><ul><li><a class="pageSidebar" href="/Team:Fudan-TSI/Team">Members</a></li><li><a class="pageSidebar" href="/Team:Fudan-TSI/Attributions">Attributions</a></li><li><a class="pageSidebar" href="http://2018.igem.org/Team:Fudan/Heritage" target=_blank>Heritage</a></li></ul></div></li>
      </ul><!-- .expandable -->
    </li>
    <li><div class="placeHolder"></div></li>
  </ul>
  </header>

  <div id="pageContent">
      <div id="contentBanner" class="figureBanner">
          <div class="row">
              <div class="col s12 hide-on-med-and-up">
                  <h1><br/>Design</h1>
                  <p class="flow-text">We hereby present a toolbox for <i>in vivo</i> continuous mutation library construction. R-Evolution could mutate coding sequences and regulatory sequences, which enables the evolution of individual proteins or multiple targets at a time, promotes high-throughput research, and serves as a foundational advance to synthetic biology.</p>
              </div>
          </div>
          <div class="hide-on-small-only">
              <div style="text-align:center;padding-top:80px"><center><img style="height:120px;width:auto" alt="cover Design" src="https://2019.igem.org/File:T--Fudan-TSI--coverDesign.gif" /></center></div>
          </div>
      </div>

<!--////////////////////////////////////////////////////
      do not edit above, if must BE CAREFUL
  //////////////////////////////////////////////////////-->
      <div class="container">
          <!-- side navigator of page content -->
          <ul id="pageContentNav" class="hide-on-med-and-down z-depth-0">
              <li class="onThisPageNav"><a href="#section1">MMLV-RT</a></li>
              <li class="onThisPageNav"><a href="#section2">tRNA&nbsp;primer</a></li>
              <li class="onThisPageNav"><a href="#section3">Flanking&nbsp;sequences</a></li>
              <li class="onThisPageNav"><a href="#section4">Cre&nbsp;recombinase</a></li>
              <li class="onThisPageNav"><a href="#section5">Experiments</a></li>
              <li class="onThisPageNav"><a href="#section6">Testing</a></li>
              <li class="onThisPageNav"><a href="/Team:Fudan-TSI/Applied_Design">Applied&nbsp;design</a></li>
              <li class="onThisPageNav"><a href="#section7">References</a></li>
              <li><a href="/Team:Fudan-TSI/Design">Design</a></li>
          </ul>
          <!-- main content of the page -->
          <main>

<div id="section1" class="section container scrolSpy">
  <h2>Moloney murine leukemia virus (MMLV) reverse transcriptase</h2>
  <h4>Why MMLV?</h4>
  <p class="flow-text">Reverse transcriptase (RT) is one of the most crucial part in our system, we chose it from Moloney murine leukemia virus (MMLV) for five reasons.</p>
  <ul>
    <li>MMLV-RT’s enhanced version is commonly used in in vitro reverse transcription, which guarantees the safety and well characterization of this part.</li>
    <li>Unlike the reverse transcriptase of HIV, MMLV-RT acts as a monomer, this brings less trouble to its production in an heterogenous host.</li>
    <li>MMLV is a eukaryotic virus, which means it’s orthogonal to prokaryotic species. This orthogonality makes up of the host adaptability of our system.</li>
    <li>MMLV-RT has a higher processivity in relation to other viruses’ RT such as HIV and AMV. This feature gives our system a greater range of mutagenesis length.</li>
    <li>MMLV-RT has low primer specificity, which means that if we change the sequence of its primer and its corresponding primer binding site (PBS), unlike HIV, the change will not be reverted in the following reverse transcription process. This makes it possible for us to customize the sequence of the tRNA primer used in accord with the target sequence in <a href="/Team:Fudan-TSI/Software">our software</a>.</li>
  </ul>
  <p class="flow-text">Even though RT does the function of reverse transcription, what is expressed in the cell is its polyprotein version. The gag-pol polyprotein has four parts—capsid protein, protease (stop codon separated), reverse transcriptase, and integrase. The integrase is deleted from the polyprotein to eliminate the possibility of genome interference. The protease contains a UAG stop codon at its 5<sup>th</sup> amino acid site <a href="#Ref1">(Yoshinaka et al.)</a>, which is readthrough as glutamine at a 5% efficiency in its native host cells, to enable the 20:1 ratio between capsid and reverse transcriptase protein. As its readthrough efficiency is much lower in <i>E. coli</i> cells, and studies have shown that lower efficiency greatly damages the activity of reverse transcriptase <a href="#Ref2">(Csibra et al.)</a>, we mutated the UAG codon into CAG, making a complete readthrough of the polyprotein. This will slightly decrease the activity of RT but within an acceptable range.</p>

  <div class="figureHolder" id="Fig1">
    <img class="responsive-img" src="https://2019.igem.org/wiki/images/4/43/T--Fudan-TSI--4mh8.gif" />
    <p>Figure 1. Crystal structure of MMLV reverse transcriptase (PDB:4MH8).</p>
  </div>

  <p class="flow-text">The capsid protein is necessary as it has been found to promote the annealing of tRNA primer to the primer binding site (PBS) in MMLV, and plays an important role in the following two strand transfer steps <a href="#Ref3">(Mak et al.</a>, <a href="#Ref4">Gonsky et al.)</a>. To be certain of this design, we have consulted <a href="/Team:Fudan-TSI/Human_Practices#IntegratedHumanPractice">Prof. Alper</a> through email and received his confirmation on the necessity of the capsid protein.</p>
  <p class="flow-text">To increase the mutation of our RT, we built a mutated version (Y1245F). This mutation has been shown to increase the mutation level by 5 times <a href="#Ref5">(Zhang et al.)</a>.</p>
  <p class="flow-text">The gag-pol polyprotein is placed under Lac operon, whose expression controlled by IPTG.</p>
</div>

<div id="section2" class="section container scrolSpy">
  <h2>tRNA primer</h2>
  <p class="flow-text">The initiation of reverse transcription requires its cognate tRNA primer. For MMLV-RT, the primer is tRNA<sup>Pro</sup>(AGG) <a href="#Ref6">(Harada et al.)</a>. After comparing its sequence to the endogenous tRNA sequences in <i>E. coli</i>, we found very little homology between them, which again proves our system’s orthogonality to prokaryotic system. However, this also arises problems regarding its post-transcriptional processing.</p>

  <div class="figureHolder" id="Fig2">
    <img class="responsive-img" src="https://2019.igem.org/wiki/images/9/9a/T--Fudan-TSI--designRT.gif" />
    <p>Figure 2.</p>
  </div>

  <p class="flow-text">To make sure that the exogenous tRNA could be successfully expressed and processed in <i>E. coli</i> cells into its matured form, we placed it under two promoters, one T7 promoter, and another pGlnW promoter, which is the endogenous promoter for GlnW and its downstream MetU. The CCA motif is added to the 3’ end, and followed by the interval sequence between two tRNA<sup>Gln</sup>, and a T7Te terminator.</p>
  <p class="flow-text">Apart from using the native tRNA<sup>Pro</sup>, which requires an additional PBS outside the target sequence, we have designed <a href="/Team:Fudan-TSI/Software">a software tool</a> which can design novel tRNAs aligning with the target sequence.</p>
</div>

<div id="section3" class="section container scrolSpy">
  <h2>Flanking sequences</h2>
  <p class="flow-text">The reverse transcriptase has three functions: 1) RNA-dependent DNA polymerase; 2) DNA-dependent DNA polymerase; 3) RNaseH activity.</p>
  <p class="flow-text">In a full reverse transcription process, the target strand will go through annealing with tRNA primer, and two strand transfer. To make sure that all three stages will be completed successfully, the noncoding sequences on the MMLV genome has been cloned to flank the target sequence. Flanking sequences include the following components, which are transcribed with the target and helps to accomplish reverse transcription.</p>
  <ul>
    <li>Primer binding site (PBS) recognizes and anneals with tRNA <a href="#Ref7">(Lund et al.)</a>, initiates reverse transcription.</li>
    <li>5’ noncoding sequence (U5) has been shown to affect the replication ability of RT and its alteration affects the synthesis of minus strand DNA <a href="#Ref8">(Kulpa et al.)</a>. It is also linked with the efficiency of the first strand transfer.</li>
    <li>Poly-purine tract (PPT) is resistant to RNaseH digestion, its remaining RNA serves as primer for the plus strand initiation <a href="#Ref9">(Finston et al.</a>, <a href="#Ref10">Kelleher et al.)</a>. We also reserved several nucleotides near its site for they have shown to impact the function of PPT <a href="#Ref11">(Robson et al.)</a>.</li>
  </ul>
</div>

<div id="section4" class="section container scrolSpy">
  <h2>Cre recombinase</h2>
  <p class="flow-text">Cre recombinase is derived from P1 bacteriophage, which could initiate recombination events between two loxP sites. Cre recombinase binds to the palindromic sequence in loxP, and after forming a tetramer (two Cre on one loxP), its active site would initiate different recombination process based on the orientation of the two loxP <a href="#Ref12">(Stark et al.)</a>. Two loxP sites in the same direction would initiate self-splicing, resulting the sequence in between be excised and form a circular loop. If the two loxP sites are oriented opposingly, the sequence in between would be inverted.</p>

  <div class="figureHolder" id="Fig3">
    <img class="responsive-img" src="https://2019.igem.org/wiki/images/f/f7/T--Fudan-TSI--3mgv.gif" />
    <p>Figure 3. Crystal structure of Cre recombinase bound to a loxP holliday junction (PDB:3MGV).</p>
  </div>

  <p class="flow-text">If the two loxP sites are placed on different sequences, the two sequence would be transferred into each other’s place, but at a lower efficiency than the other two events. Our system utilizes its strand-transfer recombination activity to insert the mutated target into its original place, thus replacing the unmutated version and allows for another round of mutation.</p>
  <p class="flow-text">The expression of Cre recombinase is placed under a different operon, and anhydrotetracycline (aTc) serves as its inducer. The reason behind placing RT and Cre under different control has been elaborated in <a href="/Team:Fudan-TSI/Model">our modeling</a>. Apart from the need of different final concentration of these two proteins for the system to achieve its optimal function, putting them under different promoters also enables better control over the system’s status.</p>
  <h4>Sequences flanking loxP</h4>
  <p class="flow-text">Cre recognizes loxP sites and after forming a tetramer, initiates the recombination process (ref12). The 2 loxP sites are placed at the farthest 5’ and 3’ end of the target sequence respectively. Since we need to utilize the recombination activity and eliminate its self-splicing ability, the two loxP sites need to be compatible with itself but incompatible with each other. We chose lox511, lox5171 and lox2272 to pair with wild-type loxP sites <a href="#Ref13">(Hoess et al.</a>, <a href="#Ref14">Lee et al.)</a>, and examined their incompatibility with wild-type loxP.</p>
  <h4>Cre mutation and alternative versions</h4>
  <p class="flow-text">We found that even though we’re putting Cre under a controllable promoter, its small leakage already can initiate self-splicing between 2 wildtype loxP (see Results for elaboration). This is undesirable as uncontrollable recombination could greatly damage the confidentiality of our result. The desired sequence could be recombined and expressed for a time and then be gone with the ongoing recombination. Our <a href="/Team:Fudan-TSI/Model">modeling</a> result also shows that Cre expression needs to stay at a low level for a higher recombination rate.</p>
  <p class="flow-text">To bring Cre expression under more stringent control, we made several different versions of Cre.</p>
  <ul>
  <li>Firstly, we mutated some of the encoding codons to rare codons in hope that this would bring difficulty to translation and thus bring down the expression level.</li>
  <li>Then, we added different degradation tags following it. We designed the tags utilizing the endogenous degradation system of <i>E. coli</i> <a href="#Ref15">(Karzai et al.)</a>. We tested 5 tags, (YA)LAA, LVA, LAV, LVV, and (WV)LAA based on research literatures <a href="#Ref16">(Landry et al.</a>, <a href="#Ref17">Janssen et al.)</a>. With the support of <a href="/Team:Fudan-TSI/Model">our modeling</a>, we found that WVLAA tag best suits our need, with moderate steady state level and quick degradation dynamic.</li>

  <div class="figureHolder" id="Fig4">
    <img class="responsive-img" src="https://2019.igem.org/wiki/images/e/ee/T--Fudan-TSI--creVersions.gif" />
    <p>Figure 4.</p>
  </div>

  <li>As an alternative approach, we also tested the split-Cre construct <a href="#Ref18">(Jullien et al.)</a>. Cre recombinase is split between its 59<sup>th</sup> and 60<sup>th</sup> amino acid, the N- and C-terminal fragments are attached to FRB and FKBP separately. When the inducer rapamycin is absent, the two fragments will not polymerize and no detectable Cre activity is found. After adding rapamycin into the culture, FKBP and FRB will polymerize, thus bringing the two Cre fragments into contact, and recombination activity will be gained.</li>
  </ul>
</div>

<div id="section5" class="section container scrolSpy">
  <h2>Experiments</h2>
  <h4>Successful induction and processing of reverse transcriptase</h4>
  <p class="flow-text">EGFP is cloned in the place of the reverse transcriptase, and fluorescence level is measured. The gag-pol polyprotein is induced and SDS-PAGE is run to see if it has been successfully processed by the protease.</p>

  <div class="figureHolder" id="Fig5">
    <img class="responsive-img" src="https://2019.igem.org/wiki/images/6/6c/T--Fudan-TSI--creFunction.gif" />
    <p>Figure 5.</p>
  </div>

  <h4>Reverse transcription completion</h4>
  <p class="flow-text">When Cre is not present in the cell, the system’s outcome would be reverse transcribed double-stranded cDNA. After lysing the cell and extracting its DNA components within, PCR amplification of the cDNA product would be performed and after electrophoresis, we could observe a brighter band if RT is induced to express compared to RT non-existent cells. For more quantitative analysis, qPCR would be performed.</p>
</div>

<div id="section6" class="section container scrolSpy">
  <h2>Testing</h2>
  <h4>Cre expression</h4>
  <p class="flow-text">EGFP is put under the ptetR promoter in place of Cre, and its fluorescence level is measured.</p>
  <p class="flow-text">Cre is co-transformed with another plasmid containing a stably expressed mCherry with two wildtype loxP sites flanking. Afterwards the whole sequence is amplified through PCR, and if Cre is in the system, mCherry would be spliced from its plasmid, and the amplification result would be a short sequence, while the non-spliced mCherry would be fully amplified.</p>
  <h4>Compatibility test of loxP sites</h4>
  <p class="flow-text">Cre is co-transformed with another plasmid containing a stably expressed mCherry flanking by two different loxP sites. The whole region is PCR amplified, and gel run to see if self-splicing event has occurred.</p>
  <h4>Degradation tag dynamic</h4>
  <p class="flow-text">EGFP carrying all five tags have been cloned and measured for its fluorescent level.</p>
  <h4>Recombination event test</h4>
  <p class="flow-text">Cre is co-transformed with two plasmids. One plasmid contains a stably expressed full-length mCherry flanked by 2 different loxP sites; the other carries a truncated version of mCherry, its flanking loxP sites match with the first plasmid. The truncated version is known to be inactive. If recombination occurs successfully, we would detect the first plasmid carrying the truncated version through PCR amplification and agarose gel analysis. Also, the fluorescent level would decrease.</p>
  <h4>System verification</h4>
  <p class="flow-text">We have constructed plasmids carrying chloramphenicol resistance gene with <a href="/Team:Fudan-TSI/Part_Collection">different stop-codon mutations</a>. The gene is thus inactive, and acts as the target sequence. The two plasmids, carrying the target and other mutation necessary components respectively, are co-transformed.</p>
  <p class="flow-text">After incubation, the reverse transcriptase would be induced first, then Cre recombinase. The target sequence would be randomly mutated in the system, and if the mutation happens to successfully reverse the stop-codon back into its original coding sequence, the cell would grow successfully on solid medium containing chloramphenicol. By counting the number of cells grown, we could get a picture of the efficiency of single site mutation.</p>

  <div class="figureHolder" id="Fig6">
    <img class="responsive-img" src="https://2019.igem.org/wiki/images/6/6c/T--Fudan-TSI--designDemo.gif" />
    <p>Figure 6.</p>
  </div>

  <h4>Sequence specificity</h4>
  <p class="flow-text">The surviving cells would be cultured and plasmid extracted. The plasmids would be sent for sequencing to prove that other sequences unrelative to the target is not mutated. It would also be retransformed into bacteria which has never encountered the system, and be planted on media containing chloramphenicol. If the cell could still grow normally, we could assume that the bacteria’s genome is not mutated in our mutagenesis process.</p>
  <h4>Host adaptability</h4>
  <p class="flow-text">Since we’re using parts that are orthogonal of native bacterial systems, our system is expected to work independently in different species. The system is tested in <i>E. coli</i>, and will then expand to species including cyanobacteria and lactic acid bacteria.</p>
</div>

<div id="section7" class="section container scrolSpy">
  <h2>References</h2>
  <ol id="ref" style="margin: 23px 0 0 0;">
      <li>[1] Yoshinaka, Y., Katoh, I., Copeland, T. D. &amp; Oroszlan, S. <a href="https://www.ncbi.nlm.nih.gov/pubmed/3885215" name="Ref1">Murine leukemia virus protease is encoded by the gag-pol gene and is synthesized through suppression of an amber termination codon.</a> <i>Proc. Natl. Acad. Sci. U. S. A.</i> 82, 1618®C1622 (1985).</li>
      <li>[2] Csibra, E., Brierley, I. &amp; Irigoyen, N. <a href="http://jvi.asm.org/content/88/18/10364.abstract" name="Ref2">Modulation of Stop Codon Read-Through Efficiency and Its Effect on the Replication of Murine Leukemia Virus.</a> <i>J. Virol</i>. 88, 10364 (2014).</li>
      <li>[3] Mak, J. &amp; Kleiman, L. <a href="https://www.ncbi.nlm.nih.gov/pubmed/9343157" name="Ref3">Primer tRNAs for reverse transcription. <i>J. Virol</i>. 71, 8087®C8095 (1997).</a></li>
      <li>[4] Gonsky, J., Bacharach, E. &amp; Goff, S. P. <a href= "https://www.ncbi.nlm.nih.gov/pubmed/11222684" name="Ref4">Identification of residues of the Moloney murine leukemia virus nucleocapsid critical for viral DNA synthesis in vivo.</a> <i>J. Virol</i>. 75, 2616®C2626 (2001).</li>
      <li>[5] Zhang, W.-H., Svarovskaia, E. S., Barr, R. &amp; Pathak, V. K. <a href="https://www.ncbi.nlm.nih.gov/pubmed/12119402" name="Ref5">Y586F mutation in murine leukemia virus reverse transcriptase decreases fidelity of DNA synthesis in regions associated with adenine-thymine tracts.</a> <i>Proc. Natl. Acad. Sci. U. S. A.</i> 99, 10090®C10095 (2002).</li>
      <li>[6] Harada, F., Peters, G. G. &amp; Dahlberg, J. E. <a href="http://www.jbc.org/content/254/21/10979" name="Ref6">The primer tRNA for Moloney murine leukemia virus DNA synthesis. Nucleotide sequence and aminoacylation of tRNAPro.</a> <i>J. Biol. Chem.</i> 254, 10979®C10985 (1979).</li>
      <li>[7] Lund, A. H., Duch, M., Lovmand, J., J?rgensen, P. &amp; Pedersen, F. S. <a href="https://www.ncbi.nlm.nih.gov/pubmed/8995641" name="Ref7">Complementation of a primer binding site-impaired murine leukemia virus-derived retroviral vector by a genetically engineered tRNA-like primer.</a> <i>J. Virol.</i> 71, 1191®C1195 (1997).</li>
      <li>[8] Kulpa, D., Topping, R. &amp; Telesnitsky, A. <a href="https://www.ncbi.nlm.nih.gov/pubmed/9049314" name="Ref8">Determination of the site of first strand transfer during Moloney murine leukemia virus reverse transcription and identification of strand transfer-associated reverse transcriptase errors.</a> <i>EMBO J.</i> 16, 856®C865 (1997).</li>
      <li>[9] Finston, W. I. &amp; Champoux, J. J. <a href="https://www.ncbi.nlm.nih.gov/pubmed/6202882" name="Ref9">RNA-primed initiation of Moloney murine leukemia virus plus strands by reverse transcriptase in vitro.</a> <i>J. Virol.</i> 51, 26®C33 (1984).</li>
      <li>[10] Kelleher, C. D. &amp; Champoux, J. J. <a href="http://www.jbc.org/content/275/17/13061" name="Ref10">RNA Degradation and Primer Selection by Moloney Murine Leukemia Virus Reverse Transcriptase Contribute to the Accuracy of Plus Strand Initiation.</a> <i>J. Biol. Chem.</i> 275, 13061®C13070 (2000).</li>
      <li>[11] Robson, N. D. &amp; Telesnitsky, A. <a href="https://www.ncbi.nlm.nih.gov/pubmed/11044073" name="Ref11">Selection of optimal polypurine tract region sequences during Moloney murine leukemia virus replication.</a> <i>J. Virol.</i> 74, 10293®C10303 (2000).</li>
      <li>[12] Stark, W. M. <a href="https://www.asmscience.org/content/journal/microbiolspec/10.1128/microbiolspec.MDNA3-0046-2014" name="Ref12">The Serine Recombinases.</a> <i>Microbiol. Spectr.</i> 2, (2014).</li>
      <li>[13] Hoess, R. H., Wierzbicki, A. &amp; Abremski, K. <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC339658/" name="Ref13">The role of the loxP spacer region in P1 site-specific recombination.</a> <i>Nucleic Acids Res.</i> 14, 2287®C2300 (1986).</li>
      <li>[14] Lee, G. &amp; Saito, I. <a href="http://www.sciencedirect.com/science/article/pii/S0378111998003254" name="Ref14">Role of nucleotide sequences of loxP spacer region in Cre-mediated recombination.</a> <i>Gene</i> 216, 55®C65 (1998).</li>
      <li>[15] Karzai, A. W., Roche, E. D. &amp; Sauer, R. T. <a href="https://www.nature.com/articles/nsb0600_449" name="Ref15">The SsrA®CSmpB system for protein tagging, directed degradation and ribosome rescue.</a> <i>Nat. Struct. Biol.</i> 7, 449®C455 (2000).</li>
      <li>[16] Landry, B. P., St?ckel, J. &amp; Pakrasi, H. B. <a href="https://www.ncbi.nlm.nih.gov/pubmed/23396339" name="Ref16">Use of degradation tags to control protein levels in the Cyanobacterium Synechocystis sp. Strain PCC 6803.</a> <i>Appl. Environ. Microbiol.</i> 79, 2833®C2835 (2013).</li>
      <li>[17] Janssen, B. D. &amp; Hayes, C. S. <a href="https://www.ncbi.nlm.nih.gov/pubmed/22243584" name="Ref17">The tmRNA ribosome-rescue system.</a>, <i>Adv. Protein Chem. Struct. Biol.</i> 86, 151®C191 (2012).</li>
      <li>[18] Jullien, N., Sampieri, F., Enjalbert, A. &amp; Herman, J.-P. <a href="https://www.ncbi.nlm.nih.gov/pubmed/14576331" name="Ref18">Regulation of Cre recombinase by ligand-induced complementation of inactive fragments.</a> <i>Nucleic Acids Res.</i> 31, e131®Ce131 (2003).</li>
  </ol>
</div>

<!--////////////////////////////////////////////////////
      do not edit below, if must BE CAREFUL
  //////////////////////////////////////////////////////-->
      </main></div><!-- end of side navigator and main of the page -->

<!--Abstract on content page-->
      <div id="abstractContent" class="z-depth-2">
          <a href="#!"><img alt="project summary" src="https://2018.igem.org/wiki/images/9/96/T--Fudan--X.svg"></a>
          <div class="container">
              <h5 style="margin:0;padding:10px 0;">Project by Team:Fudan-TSI</h5>
              <p class="flow-text" style="margin:0">Mutation library generation is critical for biological and medical research, but current methods cannot mutate a specific sequence continuously without manual intervention. We hereby present a toolbox for <i>in vivo</i> continuous mutation library construction. First, the target DNA is transcribed into RNA. Next, our reverse transcriptase (RT) reverts RNA into cDNA, during which the target is randomly mutated by our RT's enhanced error-prone ability. Finally, the mutated version replaces the original sequence through recombination. These steps will be carried out iteratively, generating a random mutation library of the target with high efficiency as mutations accumulate along with bacterial growth. Our toolbox is orthogonal and provides a wide range of applications among various species. R-Evolution could mutate coding sequences and regulatory sequences, which enables the evolution of individual proteins or multiple targets at a time, promotes high-throughput research, and serves as a foundational advance to synthetic biology.
              </p>
          </div>
      </div>
<!-- Floating Btns, Footer with sponsors -->
      <div class="floatingBtn"> <a href="#!" id="abstractBtn" class="btn"> <i class="fa fa-sticky-note" style="font-size:30px;line-height:50px"></i> </a> <a href="#FudanTSIdivWrapper" class="btn"> <i class="fa fa-angle-up" style="font-size:48px;line-height:45px"></i> </a></div> <footer id="FudanTSIfooter" class="page-footer blue-grey darken-1"><div class="container"><div class="row"><div id="sponsor" class="col m3 s12 row"> <a href="https://2019.igem.org/Team:Fudan-TSI"><img alt="2019 Team:Fudan-TSI logo white" class="col s3 m6 l3" style="position:relative; padding: 0.45em 0.3rem; margin:-0.15rem 0; left: -0.45rem;" src="https://2019.igem.org/wiki/images/0/0f/T--Fudan-TSI--LogoGrey.gif"> </a><a href="http://www.fudan.edu.cn/en/" target="_blank"><img class="col s3 m6 l3" alt="Fudan University" src="https://2018.igem.org/wiki/images/f/f7/T--Fudan--schoolLogo.png"> </a><a href="http://life.fudan.edu.cn/" target="_blank"><img class="col s3 m6 l3" style="margin-bottom: 4%;/* 该图比其他小一点，排版需要 */" alt="School of Life Sciences, Fudan University" src="https://2018.igem.org/wiki/images/1/1d/T--Fudan--schoolOfLifeSciencesIcon.png"> </a><a href="http://www.yfc.cn/en/" target="_blank"><img class="col s3 m6 l3" style="padding: 0.15rem 0.9rem;" alt="Yunfeng Capital" src="https://2018.igem.org/wiki/images/e/e2/T--Fudan--yunfengLogo.png"> </a><h3 class="col s12" style="text-align:left;font-size:12.5px">R-Evolution: an <i>in vivo</i> sequence-specific toolbox for continuous mutagenesis</h3></div><div id="footerNavList" class="col m9 s12 row"><div class="col s12 l6 row"><div class="col s12 m4 active"> <span><a href="/Team:Fudan-TSI/Description">Project</a></span><ul><li><a href="/Team:Fudan-TSI/Description">Background</a></li><li><a href="/Team:Fudan-TSI/Design">Design</a></li><li><a href="/Team:Fudan-TSI/Experiments">Experiments</a></li><li><a href="/Team:Fudan-TSI/Applied_Design">Applied design</a></li><li><a href="/Team:Fudan-TSI/Judging">Judging</a></li></ul></div><div class="col s12 m4"> <span><a href="/Team:Fudan-TSI/Demonstrate">Results</a></span><ul><li><a href="/Team:Fudan-TSI/Demonstrate#ReverseTranscription">Reverse transcription</a></li><li><a href="/Team:Fudan-TSI/Demonstrate#Recombination">Recombination</a></li><li><a href="/Team:Fudan-TSI/Demonstrate">Demonstration</a></li><li><a href="/Team:Fudan-TSI/Measurement">Measurement</a></li><li><a href="/Team:Fudan-TSI/Notebook">Notebook</a></li></ul></div><div class="col s12 m4"> <span><a href="/Team:Fudan-TSI/Model">Model</a></span><ul><li><a href="/Team:Fudan-TSI/Model">Modeling</a></li><li><a href="/Team:Fudan-TSI/Software">Software</a></li><li><a href="/Team:Fudan-TSI/Hardware">Hardware</a></li></ul></div></div><div class="col s12 l6 row"><div class="col s12 m4"> <span><a href="/Team:Fudan-TSI/Parts">Parts</a></span><ul><li><a href="/Team:Fudan-TSI/Basic_Part">Basic parts</a></li><li><a href="/Team:Fudan-TSI/Composite_Part">Composite parts</a></li><li><a href="/Team:Fudan-TSI/Improve">Part improvement</a></li><li><a href="/Team:Fudan-TSI/Part_Collection">Part collection</a></li></ul></div><div class="col s12 m4"> <span><a href="/Team:Fudan-TSI/Human_Practices">Outreach</a></span><ul><li><a href="/Team:Fudan-TSI/Public_Engagement">Public engagement</a></li><li><a href="/Team:Fudan-TSI/Human_Practices#IntegratedHumanPractice">Integrated HP</a></li><li><a href="/Team:Fudan-TSI/Collaborations">Collaborations</a></li><li><a href="/Team:Fudan-TSI/Safety">Safety</a></li></ul></div><div class="col s12 m4"> <span><a href="/Team:Fudan-TSI/Team">Team</a></span><ul><li><a href="/Team:Fudan-TSI/Team">Members</a></li><li><a href="/Team:Fudan-TSI/Attributions">Attributions</a></li><li><a href="http://2018.igem.org/Team:Fudan/Heritage" target=_blank>Heritage</a></li></ul></div><div class="col s12 m4">&nbsp;</div></div></div></div></div><div class="footer-copyright"><div class="container"><div class="contactUS row"><div class="col s12 m6 l4"><i class="fa fa-location-arrow"></i> Life Sci Bldg, 2005 Songhu Rd, Shanghai</div><div class="col s12 m6 l2"><i class="fa fa-fax"></i> +86-21-31246727</div><div class="col s12 m6 l2"><i class="fa fa-envelope-o"></i> igem@fudan.edu.cn</div><div class="col s12 m6 l4"><i class="fa fa-twitter"></i> <i class="fa fa-wechat"></i> Fudan_iGEM</div></div></div></div> </footer>
</div></div></div><!-- #pageContent #FudanTSIBody #FudanTSIdivWrapper -->

<script src="https://2019.igem.org/wiki/index.php?title=Template:Fudan-TSI/materialize.js&action=raw&ctype=text/javascript"></script><!-- Materialize 1.0.0-rc.2 -->
<script src="https://2019.igem.org/wiki/index.php?title=Template:Fudan-TSI/Fudan-js.js&action=raw&ctype=text/javascript"></script>
</body>
</html>